src/sentry/static/sentry/app/views/projectInstall/createProject.jsx

```diff
@@ -112,10 +295,123 @@ class CreateProject extends React.Component {
           : projectName,
     }));
 
-  render() {
+  renderProjectForm = (projectName, team, teams, platform, organization, inFlight) => {
+    const createProjectFormCaptured = (
+      <CreateProjectForm onSubmit={this.createProject}>
+        <div>
+          <FormLabel>
+            {organization.features.includes(NEW_PROJECT_ISSUE_ALERT_OPTIONS_FLAG)
+              ? t('Project name')
+              : t('Give your project a name')}
+          </FormLabel>
+          <ProjectNameInput>
+            <ProjectPlatformIcon monoTone platform={platform} />
+            <input
+              type="text"
+              name="name"
+              label={t('Project Name')}
+              placeholder={t('Project name')}
+              autoComplete="off"
+              value={projectName}
+              onChange={e => this.setState({projectName: e.target.value})}
+            />
+          </ProjectNameInput>
+        </div>
+        <div>
+          <FormLabel>
+            {organization.features.includes(NEW_PROJECT_ISSUE_ALERT_OPTIONS_FLAG)
+              ? t('Name')
+              : t('Assign a Team')}
+          </FormLabel>
+          <TeamSelectInput>
+            <SelectControl
+              name="select-team"
+              clearable={false}
+              value={team}
+              placeholder={t('Select a Team')}
+              onChange={choice => this.setState({team: choice.value})}
+              options={teams.map(({slug}) => ({
+                label: `#${slug}`,
+                value: slug,
+              }))}
+            />
+            <Tooltip title={t('Create a team')}>
+              <Button
+                borderless
+                data-test-id="create-team"
+                type="button"
+                icon="icon-circle-add"
+                onClick={() =>
+                  openCreateTeamModal({
+                    organization,
+                    onClose: ({slug}) => this.setState({team: slug}),
+                  })
+                }
+              />
+            </Tooltip>
+          </TeamSelectInput>
+        </div>
+        <div>
+          <Button
+            data-test-id="create-project"
+            priority="primary"
+            disabled={inFlight || !team || projectName === ''}
```

Comment:
Check if I need another condition to ensure that radio group has at least one selected choice

Reply:
Done.